.github/workflows/docker.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
+
+jobs:
+  docker:
+    uses: famedly/github-workflows/.github/workflows/docker.yml@docker-v1
+    with:
+      push: ${{ github.event_name != 'pull_request' }} # Always build, don't publish on pull requests
+      image_name: rust
```

Comment:
The workflow defaults to the github container registry, and if I understood it correctly we want to push public repositories to github, right?

This workflow call will push the container to ghcr as the famedly org with the name "rust" when pushed to main or tags, and will build only on pull requests.

Does that sound correct?

Reply:
it does sound like what we want, but I don't see where the famedly org is coming from right now? https://github.com/famedly/github-workflows/blob/main/.github/workflows/docker.yml#L9 and https://github.com/famedly/github-workflows/blob/main/.github/workflows/docker.yml#L66 read to me like it would try to push this to `ghcr.io/rust`?

Reply:
Oof yeah, you're right. The name has to be set to `famedly/docker`, I missed that and will fix it in a bit.

Reply:
Thanks!